tokio-quiche/src/http3/driver/streams.rs

```diff
@@ -74,6 +76,8 @@ impl StreamCtx {
             queued_frame: None,
             audit_stats: Arc::new(H3AuditStats::new(stream_id)),
 
+            headers_sent: false,
```

Comment:
The name is sort of bugging me. It's only need to track the whether the very first headers frame was sent or not. But like, I also won't die inside this bikeshed from paint fumes.

Reply:
Changed to `initial_headers_sent`, but now you need to approve again :P